client/rest-high-level/src/main/java/org/elasticsearch/client/SecurityClient.java

```diff
@@ -1130,4 +1132,27 @@ public Cancellable delegatePkiAuthenticationAsync(DelegatePkiAuthenticationReque
         return restHighLevelClient.performRequestAsyncAndParseEntity(request, SecurityRequestConverters::delegatePkiAuthentication, options,
                 DelegatePkiAuthenticationResponse::fromXContent, listener, emptySet());
     }
+
+
+    /**
+     * Allows a node to join to a secured cluster using the Enroll Node API.
```

Comment:
nit: `secured` -> `secure`

Reply:
It is `secured` on purpose. I wanted to say a cluster with security features enabled, not claim that such a cluster is "secure" . Maybe I'll just say "a cluster with security features enabled" :)